typescript_coding_standards.md

```diff
@@ -0,0 +1,720 @@
+By Andreea Anichiti
```

Comment:
Let's remove the Angular information.

We can keep the ones that seem to fit our use cases.